aws-s3-bucket.yml

```diff
@@ -71,6 +71,14 @@ provision:
           - eu-west-1
         pattern: ^[a-z][a-z0-9-]+$
       prohibit_update: true
+    - field_name: boc_object_ownership
+      type: string
+      details: S3 Bucket Ownership Controls (see https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html)
+      enum:
```

Comment:
Ideally I'd like to see a test that shows this enum is actually picked up (and it's not the typo eunm). Something like this should do it: https://github.com/cloudfoundry/csb-brokerpak-aws/blob/main/integration-tests/s3_test.go#L122

Reply:
Thanks George - good idea, I've added an additional test for checking all enum properties